rockcraft/commands/init.py

```diff
@@ -133,7 +133,7 @@ class InitCommand(AppCommand):
                 # uncomment this section to enable the async workers for Gunicorn.
                 #   flask-framework/async-dependencies:
                 #       python-packages:
-                #       - gunicorn[gevent]
+                #         - gunicorn[gevent]
```

Comment:
Hmm, it seems that from 133 to 134 there are 2 extra leading spaces and from 134 to 135 there are 4 extra leading spaces, should we be consistent?